### ISSUE TITLE
Updated the BM7 spec based on feedback from the workshop

### ISSUE THREAD

**Issue by stvdwtt** [usnistgov/pfhub#772]
Primary changes under consideration:

1. Remove part C (or perhaps change it to have the interfacial width of part A)
2. Look into reducing the total simulated time, parts B and C might be too computationally intensive
3. Change the description of how the temporal OOA is calculated, requesting that the approximate spatial error is subtracted off (and the same for the spatial OOA?)
4. Fix the SymPy code
5. Explain the "L" that shows up in the auto-generated C code
6. Fix the alpha->a substitution that appears in the auto-generated C code
7. Modify the early narrative to include the source term in the original equation and then solve for it (likely more intuitive than the current set up)
8. Change the requested tasks/uploads to bullets from blocks of text
9. Emphasize that convergence tests are not required for part B (and C if extant)

**Comment by wd15** [comment]
In addition, include a "Submission Guidelines" section that focuses on the data format and data naming. I think we can improve on previous submission guideline sections from other problems, which aren't complete enough.

**Comment by wd15** [comment]
Also, units for "nominal core speed" is that GHz?

**Comment by stvdwtt** [comment]
Yes, units for "nominal core speed" is in GHz. Consistent with the rest of the site, I'm thinking that the maximum memory usage should be in KB.
